supersim_test.go

```diff
@@ -21,57 +20,70 @@ const (
 	l1BlockAddress                    = "0x4200000000000000000000000000000000000015"
 )
 
-func TestGenesisState(t *testing.T) {
-	logger := oplog.NewLogger(os.Stderr, oplog.DefaultCLIConfig())
-	supersim := NewSupersim(logger, &DefaultConfig)
-	err := supersim.Start(context.Background())
-
-	defer func() {
-		err := supersim.Stop(context.Background())
-		if err != nil {
-			t.Fatalf("Failed to stop supersim: %v", err)
+type TestSuite struct {
+	t *testing.T
+
+	Cfg      *Config
+	Supersim *Supersim
```

Comment:
is there a reason that these fields should be public?

Reply:
For now they could technically be private. I made them public in the even this test suite is used in a diff package